packages/features/lend/src/lend-sidebar/lend-balance-info.tsx

```diff
@@ -56,6 +71,18 @@ export const LendBalanceInfo = ({ setInputAmount }: LendBalanceInfoProps) => {
         },
       }}
     >
+      {insufficientAllowance === true && (
```

Comment:
@matthew-garrett, FYI i moved the allowance check to here. This is more technically correct, I think,